mlir/lib/Transforms/Utils/FoldUtils.cpp

```diff
@@ -331,6 +331,42 @@ OperationFolder::tryGetOrCreateConstant(ConstantMap &uniquedConstants,
   return newIt.first->second;
 }
 
+namespace {
```

Comment:
The convention is `static` function over unnamed namespace. https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

Reply:
Done.